hi-ml-azure/src/health_azure/himl.py

```diff
@@ -942,9 +959,6 @@ def submit_to_azure_if_needed(  # type: ignore
             environment = create_python_environment_v2(
                 conda_environment_file=conda_environment_file, docker_base_image=docker_base_image
             )
-            if entry_script is None:
```

Comment:
Does this require a change in documentation (is this changing some expected behaviour)?

Reply:
updated.